chameleon/src/main/java/org/hibernate/omm/jdbc/MongodbConnection.java

```diff
@@ -1,133 +1,120 @@
 package org.hibernate.omm.jdbc;
 
+import com.mongodb.client.ClientSession;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+import org.hibernate.omm.jdbc.adapter.ArrayAdapter;
+import org.hibernate.omm.jdbc.adapter.ConnectionAdapter;
+import org.hibernate.omm.jdbc.adapter.DatabaseMetaDataAdapter;
+
 import java.sql.Array;
-import java.sql.CallableStatement;
 import java.sql.Connection;
 import java.sql.DatabaseMetaData;
 import java.sql.PreparedStatement;
-import java.sql.SQLException;
 import java.sql.SQLWarning;
 import java.sql.Statement;
 import java.util.List;
 
-import org.hibernate.omm.jdbc.adapter.ArrayAdapter;
-import org.hibernate.omm.jdbc.adapter.ConnectionAdapter;
-import org.hibernate.omm.jdbc.adapter.DatabaseMetaDataAdapter;
-import org.hibernate.omm.jdbc.exception.CommandRunFailSQLException;
-import org.hibernate.omm.jdbc.exception.NotSupportedSQLException;
-import org.hibernate.omm.jdbc.exception.SimulatedSQLException;
-
-import com.mongodb.client.ClientSession;
-import com.mongodb.client.MongoDatabase;
-import org.bson.Document;
-
 public class MongodbConnection extends ConnectionAdapter {
 
 	private final ClientSession clientSession;
-	private final MongoDatabase mongoDatabase;
+    private final MongoDatabase mongoDatabase;
```

Comment:
I just realized we're using different tab settings.  Feel free to reformat all this if you want.

Reply:
yeah, good catch! I originally intended to use some gradle pluin to protect such issue during building but later on found it is too time consuming so I cancelled it. It is a good idea to revert back its usage.